src/connector/nodejs/examples/taosBindParamSample.js

```diff
@@ -0,0 +1,82 @@
+// const TaosBind = require('../nodetaos/taosBind');
+const taos = require('../tdengine');
+var conn = taos.connect({ host: "localhost" });
+var cursor = conn.cursor();
+
+function executeUpdate(updateSql){
+    console.log(updateSql);
+    cursor.execute(updateSql);
+}
+function executeQuery(querySql){
+    
+    let query = cursor.query(querySql);
+    query.execute().then((result=>{
+        console.log(querySql);
+        result.pretty();
+    }));
+}
+
+function stmtBindParamSample(){
+    let db = 'node_test_db';
+    let table = 'stmt_taos_bind_sample';
+
+    let createDB = `create database if not exists ${db} keep 3650;`;
+    let dropDB = `drop database if exists ${db};`;
+    let useDB = `use ${db}`;
+    let createTable = `create table if not exists ${table} `+
+                    `(ts timestamp,`+
+                    `nil int,`+
+                    `bl bool,`+
+                    `i8 tinyint,`+
+                    `i16 smallint,`+
+                    `i32 int,`+
+                    `i64 bigint,`+
+                    `f32 float,`+
+                    `d64 double,`+
+                    `bnr binary(20),`+
+                    `blob nchar(20),`+
+                    `u8 tinyint unsigned,`+
+                    `u16 smallint unsigned,`+
+                    `u32 int unsigned,`+
+                    `u64 bigint unsigned);`;

```

Comment:
json type should be supported in stmt bind too.

Reply:
I will support bind json type in next pr.